### PR TITLE
chore: Comments, Spelling, and NatSpec cleanup (SC-2083)

### PR DESCRIPTION
**Pull request by deluca-mike**
# Description

Just some cleaned up comments, spelling, and NatSpec.

See: https://app.clubhouse.io/maplefinance/story/2083/maple-core-natspec-sweep-for-spelling-correct-comments

# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ] Have any features changed or been added? If yes, outline below.
- [ ] Have any events changed or been added? If yes, outline below.
- [ ] Has all documentation been updated?

# Changelog

### CODE REVIEW

File: contracts/library/PoolLib.sol [hunk 1]
```diff
@@ -133,7 +133,7 @@ library PoolLib {
         uint256 preBurnBptBal            = bPool.balanceOf(address(this));
 
         // Burn enough BPTs for liquidityAsset to cover defaultSuffered
-        bPool.exitswapExternAmountOut(
+        bPool.exitSwapExternAmountOut(
```
Comment (lucas-manuel) [inline contracts/library/PoolLib.sol:136]: This actually needs to stay as is: https://docs.balancer.finance/api/api#exitswapexternamountout

Comment (lucas-manuel) [reply contracts/library/PoolLib.sol:136]: Thats why CI is failing

Comment (deluca-mike) [reply contracts/library/PoolLib.sol:136]: Wow, balancer. How could you.

File: contracts/LateFeeCalc.sol [hunk 2]
```diff
@@ -22,8 +22,8 @@ contract LateFeeCalc {
 
     /**
         @dev    Calculates the late fee payment for a loan.
-        @param  interest Amount of interest to be used to calculate late fee for
-        @return Late fee that is charged to borrower
+        @param  interest Amount of interest to be used to calculate late fee for,
+        @return Late fee that is charged to borrower,
```
Comment (lucas-manuel) [inline contracts/LateFeeCalc.sol:26]: Should be periods not commas

File: contracts/LoanFactory.sol [hunk 3]
```diff
@@ -17,7 +17,7 @@ contract LoanFactory is Pausable {
     uint8 public constant LATEFEE_CALC_TYPE  = 11;  // Calc type of `LateFeeCalc`
     uint8 public constant PREMIUM_CALC_TYPE  = 12;  // Calc type of `PremiumCalc`
 
-    IMapleGlobals public globals;  // Interface of MapleGlobals
+    IMapleGlobals public globals; // Interface of MapleGlobals
```
Comment (lucas-manuel) [inline contracts/LoanFactory.sol:20]: We always use a double space for inline comments

File: contracts/Pool.sol [hunk 4]
```diff
@@ -82,22 +82,22 @@ contract Pool is PoolFDT {
 
     /**
         Universal accounting law: fdtTotalSupply = liquidityLockerBal + principalOut - interestSum + poolLosses
-               liquidityLockerBal + principalOut = fdtTotalSupply + interestSum - poolLosses
+                                  liquidityLockerBal + principalOut = fdtTotalSupply + interestSum - poolLosses
```
Comment (lucas-manuel) [inline contracts/Pool.sol:85]: Lets leave the equals signs aligned

**Review by lucas-manuel — CHANGES_REQUESTED** [review]
Small changes

**Review by lucas-manuel — APPROVED** [review]

**Pull request merged**